.runs/doc-critique-howto/doc_critique.md

```
@@ -0,0 +1,226 @@
+# Documentation Critique: docs/how-to/ Guides
+
+## Inputs Used
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)